codegen/projections/white_label/spec/client_spec.rb

```diff
@@ -146,12 +146,12 @@ module WhiteLabel
         end
 
         it 'uses endpoint from options' do
-          proc = proc do |context|
+          verify = proc do |context|
```

Comment:
Was this the case? If so, can you (in a separate PR I guess?) update all occurrences?

Reply:
It doesn't actually break anything - I'll switch it back to `proc` since thats what we've used everywhere.

Reply:
Ok. It could have been my use of `proc = Proc.new`. I don't remember the exact issue I had.